src/config/config.js

```diff
@@ -20,6 +20,8 @@ const Config = {
     !process.env.REACT_APP_THROW_SAVE_PROFILE_ERRORS || process.env.REACT_APP_THROW_SAVE_PROFILE_ERRORS === 'true',
   withMockedFeeds: process.env.REACT_APP_WITH_MOCKED_FEEDS === 'true',
   safariMobileKeyboardGuidedSize: process.env.REACT_APP_SAFARI_MOBILE_KEYBOARD_GUIDED_SIZE === 'true',
+  receiveUrl: process.env.REACT_APP_RECEIVE_URL,
```

Comment:
shouldn't we use a default value? Something like `PUBLIC_URL/AppNavigation/Dashboard/Send`

Reply:
yeah. 
the story specifically explains this is for production only.

Reply:
Updated